readers-frontend/src/pages/index.astro

```diff
@@ -24,5 +25,6 @@ const featuredArticlesIds: featuredHome | null =
       />
     </div>
     <BannerEdition />
+    <Muse />
```

Comment:
The component is too small on my end it's not taking up the same width as the rest of the page. I've tried to mark the empty space with red below:
<img width="1245" alt="Screenshot 2023-03-07 at 10 39 15 am" src="https://user-images.githubusercontent.com/58376513/223398820-a32c8079-2a1c-4a88-a315-dc182822075b.png">

Making it bigger will also stop Creative writing's text from wrapping

Reply:
Also the background is not taking up the full width of the page like the design